### DISCUSSION TITLE
使用Cloudflare构建0成本的AI应用

### DISCUSSION THREAD

**Discussion by giscus** [mggger/seo-learning#6] [General]
# 使用Cloudflare构建0成本的AI应用

本文指导您如何利用Cloudflare的免费服务，包括Cloudflare AI, Worker, 和 Page，来构建一个完全免费的AI应用。

https://mggg.cloud/zh/cloudflare_free_ai/

**Comment by shenweiyan** [comment]
你的 [FreeChat](https://freechat.mggg.cloud/) 中 `Text to Image` 好像生成不了图片~

**Reply by mggger** [reply]
有什么报错吗？我这边使用是正常的
<img width="466" alt="截屏2023-11-30 10 56 31" src="https://github.com/mggger/seo-learning/assets/13358284/ca5f1a11-360a-4d4a-ae76-ef18e454b61e">

**Reply by shenweiyan** [reply]
又去试了一下，现在已经正常了 >.<
